test/pylib/scylla_cluster.py

```
@@ -1224,6 +1253,45 @@ async def recycle(self) -> None:
+        while self.teardown_callbacks:
+            handle, name = self.teardown_callbacks.pop()
+            loop, callback = _parked_teardown_callbacks.pop(handle)
+            self.logger.info("Cluster %s running teardown callback %s", self, name)
+            try:
+                res = callback()
+                if asyncio.iscoroutine(res):
+                    if loop is asyncio.get_running_loop():
+                        await res
+                    else:
+                        await asyncio.wrap_future(asyncio.run_coroutine_threadsafe(res, loop))
+            except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)